site/content/docs/developing/secrets.en.md

```diff
@@ -22,9 +33,19 @@ secrets:
   GITHUB_WEBHOOK_SECRET: GH_WEBHOOK_SECRET  
 ```
 
-Once you deploy this updated manifest, your service or job will be able to access the environment variable `GITHUB_WEBHOOK_SECRET`, which will have the value of the SSM parameter `GH_WEBHOOK_SECRET`, `secretvalue1234`.
-
+Once you deploy this updated manifest, your service or job will be able to access the environment variable `GITHUB_WEBHOOK_SECRET`, which will have the value of the SSM parameter `GH_WEBHOOK_SECRET`, `secretvalue1234`.  
 This works because ECS Agent will resolve the SSM parameter when it starts up your task, and set the environment variable for you.
 
-!!! attention
-    Secrets are not supported for Request-Driven Web Services.
+#### In Secrets Manager
+Similar to SSM, first ensure that your Secrets Manager secret has the `copilot-application` and `copilot-environments` tags.  
+
+Now if you want to pass the Secrets Manager secret named `GH_WEBHOOK_SECRET` with ARN `arn:aws:secretsmanager:us-west-2:111122223333:secret:GH_WEBHOOK_SECRET-WC3PHL`, you can modify your manifest file with:
+```yaml
+secrets:
+  # (Recommended) Option 1. Referring to the secret by name.
+  GITHUB_WEBHOOK_SECRET:
+    secretsmanager: GH_WEBHOOK_SECRET
+
+  # Option 2. Referring to the secret by ARN.
+  GITHUB_WEBHOOK_SECRET: '"arn:aws:secretsmanager:us-west-2:111122223333:secret:GH_WEBHOOK_SECRET-WC3PHL"'
```

Comment:
just checking that the double quotes inside the double quotes was intentional-- doesn't seem consistent with other examples in our docs...?

Reply:
Yea it's intentional. I believe there is an issue when the value ends with "::", YAML thinks that the ARN is a key for a map so we have to explictly tell YAML "no" this is indeed a string. That's my understanding at least.

Austin dug around back in the day to figure out the root cause: https://github.com/aws/copilot-cli/issues/2199#issuecomment-824327969.